Traceback (most recent call last):
  File "/home/an920107/Documents/Programming/C++/演算法/hw1/testing_karatsuba.py", line 24, in <module>
    with open("output_karatsuba", "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'output_karatsuba'
Traceback (most recent call last):
  File "/home/an920107/Documents/Programming/C++/演算法/hw1/testing_karatsuba.py", line 24, in <module>
    with open("output_karatsuba", "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'output_karatsuba'